javascript2/readme.md

```diff
@@ -2,6 +2,6 @@
 
 | Week | Topic | Preparation | Homework | Lesson plan |
 | ---- | ----- | ----------- | -------- |------------ |
-| 1.   | Browser environment<br>DOM manipulation <br> DOM event listeners | [Preparation](week1/preparation.md) | [Homework](week1/homework.md) | [Lesson plan](week1/lesson-plan.md)|
-| 2.   | Array functions <br> [Arrow function](week2/readme.md#arrow-function) | [Preparation](week2/preparation.md) | [Homework](week2/homework.md) | [Lesson plan](week2/lesson-plan.md)|
-| 3.   | Callback function <br> Asyncronicity <br> Scope | [Preparation](week3/preparation.md) | [Homework](week3/homework.md) | [Lesson plan](week3/lesson-plan.md)|
+| 1.   | Browser environment<br>DOM manipulation <br> DOM event listeners | [Preparation](week1/preparation.md) | [Homework](/homework-projects/readme.md) | [Lesson plan](week1/lesson-plan.md)|
+| 2.   | Array functions <br> [Arrow function](week2/readme.md#arrow-function) | [Preparation](week2/preparation.md) | [Homework](/homework-projects/readme.md) | [Lesson plan](week2/lesson-plan.md)|
+| 3.   | Callback function <br> Asyncronicity <br> Scope | [Preparation](week3/preparation.md) | [Homework](/homework-projects/readme.md) | [Lesson plan](week3/lesson-plan.md)|
```

Comment:
I see the README is renamed to uppercase, but many places where it's reffered to it's used as lowercase. Would it make sense to keep it consistent, even if the linking is not case-sensitive?
```suggestion
| 3.   | Callback function <br> Asyncronicity <br> Scope | [Preparation](week3/preparation.md) | [Homework](/homework-projects/README.md) | [Lesson plan](week3/lesson-plan.md)|
```

Reply:
Oo good catch! It looks like we use lower case for readme's in the rest of this repo. I'm thinking to rename the file for consistency instead.